.github/actions/execute-assert-arc-e2e/action.yaml

```diff
@@ -23,6 +23,14 @@ inputs:
   arc-controller-namespace:
     description: 'The namespace of the configured gha-runner-scale-set-controller'
     required: true
+  wait-to-finish:
+    description: 'Wait for the workflow run to finish'
+    required: true
+    default: "true"
+  wait-to-running:
+    description: 'Wait for the workflow run to start running'
+    required: true
+    default: "false"
```

Comment:
These have been added to make the action a bit more generic. Sometimes, we just want to trigger the workflow run and not necessarily wait until it finishes successfully. These inputs allow for that.